editor/src/components/canvas/plugins/tailwind-style-plugin.ts

```diff
@@ -59,6 +60,55 @@ function getTailwindClassMapping(classes: string[], config: Config | null) {
   return mapping
 }
 
+function getStylePropContents(
+  styleProp: JSExpression,
+): Array<{ key: string; value: unknown }> | null {
+  if (styleProp.type === 'ATTRIBUTE_NESTED_OBJECT') {
+    return mapDropNulls(
+      (c): { key: string; value: unknown } | null =>
+        c.type === 'PROPERTY_ASSIGNMENT' &&
+        c.value.type === 'ATTRIBUTE_VALUE' &&
+        typeof c.key === 'string'
+          ? { key: c.key, value: c.value.value }
+          : null,
+      styleProp.content,
+    )
+  }
+
+  if (styleProp.type === 'ATTRIBUTE_VALUE' && typeof styleProp.value === 'object') {
+    return mapDropNulls(
+      ([key, value]) => (typeof key !== 'object' ? null : { key, value }),
+      Object.entries(styleProp.value),
+    )
+  }
+
+  return null
+}
+
+function getRemoveUpdates(properties: PropertyPath[]) {
```

Comment:
Some of these functions are missing return types. Can you add them please

Reply:
added them in https://github.com/concrete-utopia/utopia/pull/6549/commits/c84c9525c010928f343789d907b974835e0fdb9c